### PR TITLE
Added IP Issue no: #1746

### PR DESCRIPTION
**Pull request by ghost**
I added so when the player is banned its not just there name it is there IP address

Issue: [#1746](https://github.com/TotalFreedom/TotalFreedomMod/issues/1746)

### CODE REVIEW

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_tban.java [hunk 1]
```diff
@@ -55,6 +55,16 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
         FUtil.adminAction(sender.getName(), "Tempbanning: " + player.getName() + " for 5 minutes.", true);
         plugin.bm.addBan(Ban.forPlayer(player, sender, FUtil.parseDateOffset("5m"), reason));
 
+        // add ip ban for 5mins
+        String ip = player.getAddress().getAddress().getHostAddress();
```
Comment (OxLemonxO) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_tban.java]: You can use player.getAddress().getAddress().getHostAddress().trim();

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_tban.java [hunk 2]
```diff
@@ -55,6 +55,16 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
...
+        String[] ip_parts = ip.split("\\.");
+        if (ip_parts.length == 4)
+        {
+            ip = String.format("%s.%s.*.*", ip_parts[0], ip_parts[1]);
+        }
```
Comment (OxLemonxO) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_tban.java]: No need for this part.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_tban.java (hunk 1 above)
Comment (AvalancheYT) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_tban.java]: Or ```Ips.getIp(player)```

**Comment by JeromSar** [comment]
Doesn't `Ban.forPlayer(...)` already include the IP?

**Comment by DUFCLiam** [comment]
@Superior-tL, I shall take over this if you don't update.

**Comment by Wild1145** [comment]
This PR will remain on hold until the associated issue is reviewed, discussed and approved.

**Review by Wild1145 — CHANGES_REQUESTED** [review]
Please address the earlier comments prior to this being able to be merged.

**Pull request closed without merging**